src/components/combobox/combobox.tsx

```diff
@@ -382,6 +382,12 @@ export class Combobox implements LabelableComponent, FormComponent, InteractiveC
 
   keydownHandler = (event: KeyboardEvent): void => {
     const { key } = event;
+    const bounding = this.el.getBoundingClientRect();
```

Comment:
I think because of popper repositioning, it's the combobox that often overflows the page. This brings all of it into view. Does that work?

Reply:
I don't follow. Is `inViewport` meant to detect if the combobox is in view or the item? If it's the latter, this method needs to be updated since it's checking if the combobox (not the items) is in view, right?

Reply:
What I was trying to say, is because listbox flips, combobox is the first thing that overflows, so if you bring that in viewport, both combobox and listbox will be in view. This happens when you go up/down on list item. The only case of focusing the combobox and still having the items be out of view would be if popper didn’t flip for some reason. I wanted to make sure combobox is in view every time you interact with listbox is because you'd want to be aware of the input in it. I’ll rename the `inViewport` method so it's a bit more descriptive.